Detections/3pNetworkDetetions/PhishiglinkExecutionObserved.yaml

```diff
@@ -0,0 +1,83 @@
+id: 2fed0668-6d43-4c78-87e6-510f96f12145
+name: Phishing link click observed in 3P Network device
+description: |
+  'sucessfull Phishing link accessed via "Palo Alto Networks","Fortinet","Check Point","Zscaler" devices.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId:  OfficeATP
```

Comment:
This isn't the correct format for these check out the guidance on how to format the connectorId and dataTypes sections of the template - https://github.com/Azure/Azure-Sentinel/wiki/Query-Style-Guide#connectorid

Reply:
added separate datatypes : 

requiredDataConnectors:
  - connectorId:  OfficeATP
    dataTypes:
    - SecurityAlert
  - connectorId:  PaloAltoNetworks
    dataTypes:
    - CommonSecurityLog (PaloAlto)
  - connectorId:  Fortinet
    dataTypes:
    - CommonSecurityLog (Fortinet)
  - connectorId:  CheckPoint
    dataTypes:
    - CommonSecurityLog (CheckPoint)
  - connectorId:  Zscaler
    dataTypes:
    - CommonSecurityLog (Zscaler)